pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -176,7 +178,8 @@ func PostStartHook(hookContext genericapiserver.PostStartHookContext) error {
 		}
 
 		// ensure bootstrap rolebindings are created or reconciled
-		for _, clusterRoleBinding := range append(bootstrappolicy.ClusterRoleBindings(), bootstrappolicy.ControllerRoleBindings()...) {
+		bindNodeRole := !utilfeature.DefaultFeatureGate.Enabled(features.NodeAuthorizer)
+		for _, clusterRoleBinding := range append(bootstrappolicy.ClusterRoleBindings(bindNodeRole), bootstrappolicy.ControllerRoleBindings()...) {
```

Comment:
Yuck.  Applying filters may be more palatable since it won't lead to flag madness and the role names have to stay pretty stable. No option is pretty.